File: 2023-2024/sourcecode/mod1/lab4/22BottomNavigationBasicsJava/app/src/main/java/com/example/a22_bottomnavigationbasics_java/SecondFragment.java
package com.example.a22_bottomnavigationbasics_java;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.a22_bottomnavigationbasics_java.databinding.FragmentSecondBinding;

public class SecondFragment extends Fragment {

    private FragmentSecondBinding binding;

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = FragmentSecondBinding.inflate(inflater);
        return binding.getRoot();
    }
}

File: 2023-2024/sourcecode/mod1a/lab3/21BottomNavigationBasicsKotlin/app/src/main/java/com/example/a21_bottomnavigationbasics_kotlin/SecondFragment.kt
package com.example.a21_bottomnavigationbasics_kotlin

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.a21_bottomnavigationbasics_kotlin.databinding.FragmentSecondBinding

class SecondFragment : Fragment() {

    private lateinit var binding: FragmentSecondBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentSecondBinding.inflate(inflater)
        return binding.root
    }
}